src/brpc/controller.h

```diff
@@ -340,7 +340,7 @@ friend void policy::ProcessThriftRequest(InputMessageBase*);
     // and controller.cpp to know more.
     void allow_done_to_run_in_place()
     { add_flag(FLAGS_ALLOW_DONE_TO_RUN_IN_PLACE); }
-    // True iff above method was called.
```

Comment:
This is not a typo, ``iff`` means if and only if

Reply:
O, I see.